source/buildings/university.md

```diff
@@ -31,6 +31,8 @@ As you level up the University, you can hire more researchers:
 | 4 | 4 |
 | 5 | 5 |
 
+Additionally, offline research time is unlocked at University level 3.
```

Comment:
This needs to be elaborated a bit more in terms of what offline research is and what it does.